it/src/main/java/org/corfudb/universe/node/CorfuClient.java

```diff
@@ -0,0 +1,21 @@
+package org.corfudb.universe.node;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+/**
+ * Represent a Corfu client implementation of {@link Node}.
+ */
+public interface CorfuClient extends Node {
+    ClientParams getParams();
+
+    @Builder
+    @Getter
+    @EqualsAndHashCode
+    class ClientParams implements NodeParams {
+        private final String host;
+        private final String name;
+        private final NodeType nodeType = NodeType.CORFU_CLIENT;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)